sound/soc/sof/ops.h

```diff
@@ -21,6 +21,15 @@
 #define sof_ops(sdev) \
 	((sdev)->pdata->desc->ops)
 
+#define sof_ipc_ops(sdev) \
```

Comment:
Should we rename this to sof_dsp_ipc_ops? And all the rest too for consistency? Because my PR also adds the top-level ipc ops

Reply:
sure, this name is better. will change.

Reply:
done